src/transformers/models/maest/feature_extraction_maest.py

```diff
@@ -0,0 +1,148 @@
+# coding=utf-8
+# Copyright 2023 The HuggingFace Inc. team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Feature extractor class for Music Audio Efficient Spectrogram Transformer.
+"""
+
+
+import numpy as np
+import torch
+from librosa.feature import melspectrogram
```

Comment:
We have an `audio_utils` file that allows you to run Mel spectrograms without a `librosa` requirement. Here's an example for the SpeechT5 model: https://github.com/huggingface/transformers/blob/9ed538f2e67ee10323d96c97284cf83d44f0c507/src/transformers/models/speecht5/feature_extraction_speecht5.py#L161

In `transformers`, we try to avoid extra dependencies (like `librosa`) unless seriously required. Here, I think we can bypass the requirement by using `audio_utils`

Reply:
Addressed in 57324bf